src/main/java/cosminsechelmain/driver/BrowserManager.java

```diff
@@ -0,0 +1,12 @@
+package cosminsechelmain.driver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BrowserManager {
```

Comment:
No need to create another package with this class. You already have the one from homework 5. Remove this one and use that one instead. Thanks!